packages/server/modules/core/repositories/users.ts

```diff
@@ -51,16 +51,30 @@ export async function getUsers(
   userIds = isArray(userIds) ? userIds : [userIds]
 
   const q = Users.knex<UserWithOptionalRole[]>().whereIn(Users.col.id, userIds)
+  q.leftJoin(
+    USER_EMAILS_TABLE_NAME,
+    `${USER_EMAILS_TABLE_NAME}.userId`,
+    Users.col.id
+  ).where({ primary: true })
+
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
I assume you want all columns besides email and verified, that's why you're doing this? In that case just use lodash's `omit()`. An approach that requires you to disable eslint rules, while clever maybe, is not very clean